packages/babel-highlight/src/index.ts

```diff
@@ -76,7 +77,7 @@ function getTokenType(match) {
 /**
  * Highlight `text` using the token definitions in `defs`.
  */
-function highlightTokens(defs: Object, text: string) {
+function highlightTokens(defs: any, text: string) {
```

Comment:
Wouldn't is be better to use this here? So that it disallows primitive types.
```suggestion
function highlightTokens(defs: object, text: string) {
```

Reply:
this is part of automated conversion, the thing is that `Object` in flow are not an object… it is just an alias to `any` and so replacing it with `object` in general case is not safe

in particular case - `object` looks slightly better, but not much… eventually this should be `Record<string, ChalkType>` but thats is kinda outside of migrating existing types, and so - I was considering it as something to be done separately when reviewing and fixing all the `any`s